apps/trustlab/src/components/Card/Card.js

```diff
@@ -48,7 +49,7 @@ function Card({
           borderRadius: 0,
           border: "1px solid",
           borderColor: neutral[300],
-          ...CardProps?.sx,
+          ...sx,
```

Comment:
Shouldn't hold this PR back but lets use the `Array` form of `sx` whenever we can